### PR TITLE
K8s autoscaler

### PR DESCRIPTION
**Pull request by rueian**

### CODE REVIEW

File: packages/autoscaler/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@kubernetes-models/autoscaler",
+  "version": "1.5.2",
```
Comment (tommy351) [inline packages/autoscaler/package.json:3]:
```suggestion
  "version": "0.0.0",
```

**Review by tommy351 — COMMENTED** [review]
- Please remove the `packages/autoscaler/CHANGELOG.md`.
- Maybe you can use the YAML files from the repository directly. https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/deploy

File: packages/autoscaler/package.json (hunk 1 above)
Comment (rueian) [reply packages/autoscaler/package.json]: 👌

File: packages/autoscaler/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      name: 'my-app'
+    },
+    updatePolicy: {
+      updateMode: 'Off'
+    }
+  }
+});
+
+// Validate against JSON schema
+route.validate();
```
Comment (tommy351) [inline packages/autoscaler/README.md:32]:
```suggestion
vpa.validate();
```

Comment (rueian) [reply packages/autoscaler/README.md]: 🙆‍♂️

**Comment by tommy351** [comment]
Published as `@kubernetes-models/autoscaler@0.1.0`

**Pull request merged**